charts/brigade-github-app/templates/deployment.yaml

```diff
@@ -32,6 +32,10 @@ spec:
         type: github-app
     spec:
       serviceAccountName: {{ $serviceAccount }}
+      {{ if .Values.github.affinity }}
```

Comment:
I'd favor config being at the root level, so `.Values.affinity` and leaving GH-specific values under `.Values.github.*`